plugins/catalog/src/js/repositories/components/AddRepositoryFormModal.js

```diff
@@ -57,12 +57,20 @@ class AddRepositoryFormModal extends React.Component {
       {
         fieldType: "number",
         name: "priority",
-        placeholder: "Priority",
+        placeholder: i18n._(t`Priority`),
         required: false,
         min: "0",
         max: `${numberOfRepositories}`,
         step: "1",
-        validationErrorText: `Must be a positive integer between 0 and ${numberOfRepositories} representing its priority. 0 is the highest and ${numberOfRepositories} denotes the lowest priority.`,
+        validationErrorText: i18n._(
```

Comment:
I don't think this can be translated this way since you'll get a difference "id" phrase for every `numberOfRepositories`

Reply:
I think I was wrong about this? The phrase looks extracted.